jwst/cube_build/ifu_cube.py

```diff
@@ -221,7 +221,7 @@ def define_cubename(self):
                 fg_name = fg_name.lower()
                 newname = self.output_name_base + fg_name + '_s3d.fits'
                 if self.output_type == 'single':
-                    newname = self.output_name_base + fg_name + '_single_s3d.fits'
+                    newname = self.output_name_base + fg_name + '-single_s3d.fits'
                 if self.coord_system == 'internal_cal':
                     newname = self.output_name_base + fg_name + '_internal_s3d.fits'
```

Comment:
If the "single" file names are being changed to use a hyphen as the separator, instead of underscore, should the "internal" names be changed the same way here?